artifactory/commands/container/pull.go

```diff
@@ -34,8 +39,7 @@ func (pc *PullCommand) Run() error {
 	}
```

Comment:
I think that the following check is missinbg in this function. I copied it from the "docker push" command - 
```
	toCollect, err := pc.buildConfiguration.IsCollectBuildInfo()
	if err != nil {
		return err
	}
	if !toCollect && !pc.IsDetailedSummary() {
		return nil
	}
```
There's no detailed summary for "docker pull", so the code may need to change a bit.